README.md

```diff
@@ -86,15 +91,46 @@ div { background: red; }
 ```
 ```js
 import styles from "./styles.css"
+
+console.log(styles) // div { background: red; }
+```
+
+### Babel
```

Comment:
Babel is not part of Bundler CLI but as a plugin that can be used with Bundler API. So maybe a plugin section for babel and text plugins.